gui/index.html

```diff
@@ -97,9 +97,10 @@
 		<!-- ADDING CUSTOM BLOCKS -->
 	</xml>
 
-	<!-- basic imports -->
+	<!-- basic imports (NEED TO BE LOCALIZED) -->
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
 	<script src="http://pixijs.download/release/pixi.min.js"></script>
```

Comment:
If I'm not mistaken, we won't need Pixi.js if we've completely ported to the new gridview in d3.